ansible/roles/letsencrypt/templates/haproxy-reload.sh.j2

```diff
@@ -7,7 +7,9 @@ for domain in {{ letsencrypt_domains | join(' ') }}; do
     cert_path="/etc/haproxy/certs.d/haproxy.pem"
     # Get the full text of the certificate, deleting any blank lines (OpenSSL doesn't like those)
     full_cert=$(cat $le_base/$domain/fullchain.pem $le_base/$domain/privkey.pem | sed '/^[[:blank:]]*$/ d')
-    # Start a transaction to update the certificate
+    # Copy the cert to haproxy's config dir
+    echo $full_cert | tee $cert_path /etc/haproxy/haproxy.pem
```

Comment:
same issue as above, this writes into a config dir, but will get overwritten when cc-ansible runs.